src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConfiguredAsyncDisposable.cs

```diff
@@ -18,6 +18,8 @@ internal ConfiguredAsyncDisposable(IAsyncDisposable source, bool continueOnCaptu
             _continueOnCapturedContext = continueOnCapturedContext;
         }
 
+        /// <summary>Asynchronously releases the unmanaged resources used by the <see cref="T:System.Runtime.CompilerServices.ConfiguredAsyncDisposable" />.</summary>
+        /// <returns>A task that represents the asynchronous dispose operation.</returns>
```

Comment:
It's not actually a task.  But I see this matches what we have documented, so, fine for now.

Reply:
I've seen other places where we say "A task that represents" when the returned value is the result of a ConfigureAwait. 

For future reference, what description would you prefer to use in plain English for these cases? Would this work?:

"An awaitable, configured value task that represents..."

Or would you use something else?